cg/server/endpoints/cases.py

```diff
@@ -62,7 +68,7 @@ def get_cases_delivery_message():
             delivery_message_request
         )
         return jsonify(response.model_dump()), HTTPStatus.OK
-    except (CaseNotFoundError, OrderMismatchError) as error:
+    except (CaseNotFoundError, OrderMismatchError, CgDataError) as error:
```

Comment:
What would be the consequence of excepting all here @islean ?

Reply:
If we get a connection issue when querying the database or if something else completely unforeseen happens, we would return a bad request status. I think we should handle all the errors which we have control over. So if we know that we want a BAD REQUEST status if we can not find a case, then do so. If we can not write down more explicitly what goes wrong than a general exception, I think it should result in an internal server error. 

In general the 4XX HTTP status codes are reserved for errors done by the user, while 5XX codes are errors on our end.